.github/workflows/ci_benchmarks_macos.yaml

```diff
@@ -19,6 +19,14 @@ env:
 jobs:
   prologue:
     name: prologue
+    if: |
+      contains(github.event_name, 'pull_request')
+      || contains('refs/heads/master,
+                      refs/heads/trying,
+                      refs/heads/staging', github.ref)
+      || (github.ref == 'refs/heads/develop' && github.event.head_commit.author.name != 'bors[bot]')
+      || startsWith(github.ref, 'refs/heads/rc/')
+      || (github.repository_owner != 'nervosnetwork')
```

Comment:
btw, I prefer to replace `contains` with the equal mark `=`. I try to refactor this block to make it more readable:

```yaml
if: |
    github.repository_owner != 'nervosnetwork' ||
    githu.event_name != 'push' ||
    (
        githu.event_name == 'push' &&
        (
            github.ref == 'refs/heads/master' ||
            github.ref == 'refs/heads/trying' ||
            github.ref == 'refs/heads/staging' ||
            (github.ref == 'refs/heads/develop' && github.event.head_commit.author.name != 'bors[bot]') ||
            startsWith(github.ref, 'refs/heads/rc/')
        )
    )
```

Reply:
yes,you are right, thanks for your remanding.